x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -982,4 +982,26 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'windows.advanced.memory_protection.context_manipulation_detection',
+    first_supported_version: '8.4',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.windows.advanced.memory_protection.context_manipulation_detection',
+      {
+        defaultMessage:
+          'Detect injection based on thread context manipulation (e.g. `SetThreadContext`) as a part of memory protection. Default: true',
```

Comment:
```suggestion
          'Detect injection based on thread context manipulation (e.g. SetThreadContext) as a part of memory protection. Default: true',
```

I'm not sure how backticks will render in the tooltip.